java/test/jmri/jmrix/openlcb/configurexml/ProtocolOptionsPersistenceTest.java

```diff
@@ -187,14 +174,5 @@ public void testManyOptions() throws IOException,
         expectOptionValue("Single", "Foo", "");
     }
 
-    @Test
-    public void testLccBuffer() throws Exception {
```

Comment:
If this text is emitting an error message, and the error message is valid, then use the methods in 
https://github.com/JMRI/JMRI/blob/master/java/test/jmri/util/JUnitAppender.java to either suppress the error or asser that the error should be there.

If the error isn’t valid, use an Ignore annotation ( with a description of why we are ignoring the test ) to suppress running the test.

Reply:
I'm not quite sure what the error is (I could not find in the log where it shows the error message; just the failure claiming that there was an error message).

But actually I would need something else in order to make this test useful. Right now I cannot instantiate the LCC buffer object, because it requires a working serial port in the configuration. I don't know what I could set the serial port configuration to in order for it to work in both CI as well as developer's environments. If you have any idea, that would make this test actually useful -- as of now it was not actually testing anything, hence I deleted it.

Reply:
One trick we use sometimes is to just override any methods that actually do something with the port.  

If you put it back in with an ignore that says “needs serial port”, I can take a look at it in more detail later.

Reply:
I can't override things here, because during profile load it is the XML parser code that instantiates the class. So I cannot replace it with an anonymous class anymore.

I refactored the code to helper classes so that there is minimal code left in the actual adapter class. This way we do get test coverage via the loopback adapter (the rest of the test cases here use the loopback adapter).